src/pages/TaskRun/MappedTaskRuns-Tile.vue

```diff
@@ -75,7 +76,11 @@ export default {
       return `%${this.searchTerm}%`
     }
   },
-  methods: {},
+  methods: {
+    isFinished(state) {
+      return FINISHED_STATES.includes(state)
+    }
```

Comment:
let's make this a utility function in an external file so it can be reused in both components.